reana_job_controller/version.py

```diff
@@ -14,4 +14,4 @@
 
 from __future__ import absolute_import, print_function
 
-__version__ = "0.7.3"
+__version__ = "0.7.4rc1"
```

Comment:
BTW is our Python-tag-to-Docker-semver logic handling RC versions nicely? It will be a nice test. 

(When in doubt, we could also go with the usual `0.7.4a1` especially if may happen to change some features WRT limits.)

Reply:
Yes:
```
$ reana-dev git-create-release-commit -c r-j-controller --version 0.7.4-rc.1
$ reana-dev git-tag -c r-j-controller
$ reana-dev release-docker -c r-j-controller
[2021-03-25T14:16:10] reana-workflow-controller: git describe --tags --abbrev=0
[2021-03-25T14:16:10] reana-workflow-controller: docker tag reanahub/reana-workflow-controller:latest reanahub/reana-job-controller:0.7.4-rc.1
[2021-03-25T14:16:10] reana-job-controller: docker push reanahub/reana-job-controller:0.7.4-rc.1
The push refers to repository [docker.io/reanahub/reana-job-controller]
....
```

Should I change to `alpha.1`?